### PR TITLE
Merge Dev to Master

### PR DESCRIPTION
**Pull request by undercase**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -14,5 +15,24 @@ export default class BaseCRM extends Activity {
     @singleS3StreamOutput('json')
     async onTask(activityTask, config) {
         throw new Error('onTask not implemented');
```
Comment (perryao) [inline lib/index.js]: @undercase will want to get rid of this error

File: test/test-config.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const config = {
-
+  accessToken: "d4ba5adc6810a41fe7826c40c305b9c5f85852aba001de6ede17d57430aee0b0"
```
Comment (perryao) [inline test/test-config.js]: is this a real accessToken?

**Comment by benjamingregory** [comment]
@undercase Please confirm when you've seen @trong-le's notes.

**Comment by undercase** [comment]
I've seen and implemented his notes in the latest commit to dev.

**Pull request merged**